.babelrc

```diff
@@ -1,89 +1,21 @@
-{ 
-  "plugins": [
-    "transform-es5-property-mutators",
-    "transform-es3-member-expression-literals",
-    "transform-es3-property-literals"
+{
+  "presets": [
+    "es2015-rollup"
```

Comment:
I think there might be more than that now. I'd rather let rollup handle the confit specifics of rollup though.